src/base/utils/misc.h

```diff
@@ -103,5 +103,17 @@ namespace Utils::Misc
         const std::wstring path = (windowsSystemPath() / Path(source)).toString().toStdWString();
         return reinterpret_cast<T>(::GetProcAddress(::LoadLibraryW(path.c_str()), funcName));
     }
+
+    bool isTorrentFileAssocSet();
```

Comment:
`Utils::Misc` getting too crowded, don't you think? I would put such OS-specific utilities somewhere like `Utils::OS`.

Reply:
I'm afraid `Utils::OS` would then be crowded too as other helper functions would be eligible into this namespace.
How about `Utils::Associate` or `Utils::ContentHandler`?

Reply:
> How about `Utils::Associate` or `Utils::ContentHandler`?

Okay, better leave it as it is.

Reply:
> I'm afraid `Utils::OS` would then be crowded too as other helper functions would be eligible into this namespace.

In fact, I don't see any problems moving those functions that have a more explicit affiliation to more particular scope than `Misc`. But I'm still against too much fragmentation.

Reply:
>I would put such OS-specific utilities somewhere like Utils::OS.

Done. Maybe it could be useful.